src/routes/lib.js

```diff
@@ -186,3 +186,8 @@ export const takeScreenshot = (frameId, fileNames, elementNamesInIframe, useElem
       });
   });
 };
+
+export const toFixed = (num, digits) => {
```

Comment:
What's the reason for using this instead of the built in `toFixed()` method?

Assuming we're keeping it, I would like this function to be renamed so it's more obvious that it is different from the built in.

Reply:
The difference is that this returns a number instead of a string.

Reply:
What's wrong with using `parseFloat()` to do the conversion from string to floating point? Is there a rounding problem that you're trying to avoid?

Reply:
> What's wrong with using `parseFloat()` to do the conversion from string to floating point?

What conversion from string to floating point? There are no strings involved here. I guess you might mean if I used the builtin toFixed? But, in that case, I'd need to write `Number.parseFloat((... long calculation ...).toFixed(4))` everywhere, at which point I'd want to abstract that away into a utility function.

Reply:
Alright, just change the name to something like `toFixedAsNumber`, or maybe `customToFixed', and we can get this merged.